### DISCUSSION TITLE
Change the way "Ian Destination" is displayed in the exporter rule

### DISCUSSION THREAD

**Discussion by albertosaez** [dcm4che/dcm4chee-arc-light#2918] [Q&A] [answered]
When a large number of nodes configured, thy are listed in the the "Ian Destination" section of the exporter rule, making difficult to read the rules. 

Could be possible to present it in the same way of "Exporter ID" ?

![image](https://user-images.githubusercontent.com/3266240/102015692-63c46400-3d5d-11eb-95c1-891eda11dc00.png)

**Comment by vrindanayak** [comment]
@albertosaez : The Exporter ID is a single value field, whereas IAN destination is multi valued and hence you see a checkbox against each of the destination names. The purpose of keeping this multi valued is to enable sending out IAN notifications to multiple IAN destinations on a single export transaction. @shral may have some ideas on how this could be improved in the UI.

**Reply by albertosaez** [reply]
Hi, I think "Exported ID" could be muli-valuated

![image](https://user-images.githubusercontent.com/3266240/102091023-84082780-3e1e-11eb-83fa-7c05625de44a.png)

**Comment by shral** [comment, marked as answer]
Pushed a [change](https://github.com/dcm4che/dcm4chee-arc-light/issues/3048) regarding any dynamic field in the configuration, if there are more than 5 elements it would show like in the `Exporter ID` with the search field and scrollbar.
